### PR TITLE
update character-set-and-collation

### PR DESCRIPTION
**Pull request by wjhuang2016**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
LGTM
@bb7133 @kissmydb PTAL

File: character-set-and-collation.md [hunk 1]
```diff
@@ -51,11 +57,11 @@ SHOW COLLATION WHERE Charset = 'utf8mb4';
 2 rows in set (0.00 sec)
 ```
 
-## 集群的字符集和排序规则
+## 各个层次的字符集和排序规则
```
Comment (kissmydb) [inline character-set-and-collation.md:60]:
```suggestion
## 不同范围的字符集和排序规则
```

File: character-set-and-collation.md [hunk 2]
```diff
@@ -6,8 +6,14 @@ aliases: ['/docs-cn/dev/reference/sql/characterset-and-collation/','/docs-cn/dev
 
 # 字符集和排序规则
 
+本文介绍了 TiDB 中支持的字符集和排序规则。
+
+## 字符集和排序规则的概念
+
 字符集 (character set) 是符号与编码的集合。排序规则 (collation) 是在字符集中比较字符的规则。
```
Comment (scsldb) [inline character-set-and-collation.md:13]: 建议分成两行

File: character-set-and-collation.md [hunk 3]
```diff
@@ -6,8 +6,14 @@ aliases: ['/docs-cn/dev/reference/sql/characterset-and-collation/','/docs-cn/dev
...
```
Comment (scsldb) [inline character-set-and-collation.md:14]: 建议将所有的排序规则都罗列出来

**Comment by TomShawn** [comment]
@wjhuang2016 Please address the comments above.

**Comment by TomShawn** [comment]
@scsldb PTAL again.

**Pull request merged**